perf/benchmark/runner/runner.py

```diff
@@ -174,28 +169,50 @@ def run(self, conn, qps, size, duration):
         if self.extra_labels is not None:
             labels += "_" + self.extra_labels
 
-        grpc = ""
+        # TODO(oschaaf): For test, remove
+        duration = 1
+        
+        # Note: Labels is the last arg, and there's stuff depending on that.
+        # watch out when moving it.
+        nighthawk_args = [
+            "nighthawk_client", 
+            "--concurrency auto",
+            "--output-format json",
+            "--prefetch-connections",
+            "--open-loop",
+            "--experimental-h1-connection-reuse-strategy lru",
+            "--experimental-h2-use-multiple-connections",
+            "--nighthawk-service 127.0.0.1:{portmap}",
+            "--label Nighthawk",
+            "--connections {conn}",
+            "--rps {qps}",
+            "--duration {duration}",
+            "--request-header \"x-nighthawk-test-server-config: {{response_body_size:{size}}}\"",
+            "--label {labels}"
+        ]
+
+        # Our "gRPC" mode actually means:
+        #  - h2 
```

Comment:
Is this h2c?

Reply:
No, if mode is set to `grpc`, we'll switch to `https`. Added a code-level comment about that to clarify.